src/App.tsx

```diff
@@ -1,24 +1,27 @@
-import React from 'react';
-import logo from './logo.svg';
 import './App.css';
 
+import { useState } from 'react';
+import TodoList from './Components/ToDoList';
+import { ToDoInterface } from './Shared/interface';
+import { sampleDataForList } from './Shared/sample';
+
 function App() {
+  const toDoArray:ToDoInterface[] = sampleDataForList;
+  const [todos, setTodos] = useState(toDoArray);
+
+  const markCompleted = (id:number) => {
+    setTodos((prevTodos:ToDoInterface[]) =>
```

Comment:
don't use the interface key in the name you can use ITodos.
it will indicate interface of todos